audio/softsynth/eas.cpp

```diff
@@ -84,37 +84,30 @@ class MidiDriver_EAS : public MidiDriver_MPU401, Audio::AudioStream {
 
 private:
 	struct EASLibConfig {
-		uint32 version;
-		uint32 debug;
-		int32 voices;
-		int32 channels;
-		int32 rate;
-		int32 bufSize;
-		uint32 filter;
-		uint32 timeStamp;
+		unsigned long version;
+		unsigned debug;
+		long voices;
+		long channels;
+		long rate;
+		long bufSize;
+		unsigned filter;
```

Comment:
Where did you find the types? And can you please use ScummVM types?

Reply:
The types come from here: https://android.googlesource.com/platform/external/sonivox/+/refs/heads/master/arm-wt-22k/host_src/eas_types.h#110

The structure is a copy of `S_EAS_LIB_CONFIG`, and is passed to the dynamically loaded EAS functions. As such, the structure should be using the types that EAS is expecting.